src/content/decentralized-identity.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Decentralized Identity 
+description: An overview of decentralized identity on Ethereum
+lang: en
+template: use-cases
+emoji: ":id:"
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Centralized identity systems are fragmented, poorly secured, and offer minimal privacy protection
+summaryPoint2: Decentralized identity increases personal control of one's identity information.
+summaryPoint3: Decentralized ID projects built on Ethereum can revolutionize identity management. 
+---
+
+# What are identity, identifiers and attestations? {#what-are-identity-identifiers-attestations}
+
+## What is identity {#what-is-identity}
+
+Identity means an individual's sense of self, defined by a set of unique characteristics. Identity refers to the fact of being an _individual_, i.e., a distinct human entity. Identity could also refer to other non-human entities, such as an organization or authority.
+
+## What are identifiers? {#what-are-identifiers}
+
+An identifier is a piece of information that can be used to distinguish a person's identity. Common identifiers include the following: 
+- Name
+- Social security number/tax ID number 
+- Mobile number 
+- Date and place of birth
+- Digital identification credentials, e.g., email addresses, usernames, avatars
+
+It is important to understand that in these traditional examples of identifiers that _issuance_ and _control_ over these identifiers are maintained by a central agency. For example, legal names require permission from a government for issuance or before being changed, and changes to your social media handle require permission from the platform. 
+
+## What are attestations? {#what-are-attestations}
+ 
+An attestation refers to a claim made by an entity about another entity. An example attestation is a driver's license issued by the Department of Motor Vehicles (DMV). A driver's license, which provides a specific claim (that the holder has legal permission and is competent enough to drive a car) is made by one entity (the DMV) about another entity (the driver). 
+
+Attestations are different from identifiers. An attestation _contains_ identifiers to reference a particular identity, and makes a claim about an attribute related to this identity. The driver's license mentioned earlier, for example, has a mix of both identifiers (such as name, date of birth, and address), as well as an attestation about this individuals legal right to operate a vehicle.
```

Comment:
I think @wackerow made that edit to better highlight the difference between attestations, identifiers, and identity. Here's what I had in the original page:

*"Although attestations and identities are different, an attestation contains identifiers that point to an individual's identity. The driver's license mentioned earlier, for example, may have identifiers, like the holder’s name, date of birth, and nationality.*"

Tbh, I doubt we can simplify that concept any further. Especially because decentralized identity can be a hard concept to wrap one's head around (and we don't want readers to miss the point). Let me know what you think, though.